### ISSUE TITLE
scale learning rate

### ISSUE THREAD

**Issue by zdgithub** [horovod/horovod#1542]
**Environment:**
1. Framework: (TensorFlow, Keras, PyTorch, MXNet)
2. Framework version:
3. Horovod version:
4. MPI version:
5. CUDA version:
6. NCCL version:
7. Python version:
8. OS and version:
9. GCC version:

**Checklist:**
1. Did you search issues to find if somebody asked this question before?
2. If your question is about hang, did you read [this doc](https://github.com/horovod/horovod/blob/master/docs/running.rst)?
3. If your question is about docker, did you read [this doc](https://github.com/horovod/horovod/blob/master/docs/docker.rst)?
4. Did you check if you question is answered in the [troubleshooting guide](https://github.com/horovod/horovod/blob/master/docs/troubleshooting.rst)?

**Your question:**
Please ask your question here.
I trained my models on a GPU with batch_size=150 and lr=0.001, but now I want to train it with 3 GPUs by horovod with batch_size=50 on per GPU. To keep the similar learning curve, shall I scale the lr by 1/3? I am not clear about the reason why the DistributedOptimizer learning rate should scale up by hvd.size(), for instance, `AdamOptimiezer(0.1 * hvd.size())`, the lr of each work is `0.1` or `0.1*hvd.size()`?

**Comment by ebarsoum** [comment]
Hi @zdgithub 

If you reduced your batch size by 3, then you don't need to scale your LR. Because your effective batch size still 150. However, reducing your batch size will reduce your throughput. It is usually better to keep you batch size and scale your LR (as long as you still converge). If you kept your batch size to 150 per GPU, then your effective batch size is 450.  

Thanks,
Emad

**Comment by zdgithub** [comment]
@ebarsoum Thank you. I get it.
